cypress/e2e/news.cy.js

```diff
@@ -0,0 +1,24 @@
+describe("Work & Co News links test", () => {
+  it("Opens W&C site/News, iterates and opens each link, checks H1 for each page", () => {
+    cy.visit("https://work.co/");
+    cy.get('[data-test-id="global-menu-btn"]').click();
+    cy.get('[data-test-id="grid-news-link"]').click();
+
+    cy.get(".TitleDateList>li").then((newsLinks) => {
+      cy.wrap(newsLinks).each((newsLink, index) => {
+        if (index === 0) {
```

Comment:
This only tests first link, try to make it so that it test all links in the section

Reply:
Really? I've been running tests in Cypress UI and I see that it is opening and checking the headings for each link. Am I missing something?